main.go

```diff
@@ -425,6 +440,19 @@ func (z *Config) Archive(ctx context.Context, meeting zoom.Meeting, params runPa
 		}
 		curArchMeeting.fileNumber++
 		z.logger.Printf("uploaded %q to %s/%s", name, parent.Name, meetingFolder.Name)
+		uploaded = true
+	}
+	if uploaded && action.Slack != "" && z.slackClient != nil {
+		slackSpan, ctx := apm.StartSpan(ctx, "slack", "app")
+		body := fmt.Sprintf("%s recording now available: https://drive.google.com/drive/folders/%s", meeting.Topic, meetingFolder.Id)
+		channel, _, text, err := z.slackClient.SendMessage(action.Slack, slackapi.MsgOptionText(body, true))
```

Comment:
```suggestion
		channel, _, text, err := z.slackClient.SendMessageContext(ctx, action.Slack, slackapi.MsgOptionText(body, true))
```

then you can instrument the underlying HTTP requests